writer.go

```diff
@@ -218,20 +217,17 @@ func (p *MasterPlaylist) Encode() *bytes.Buffer {
 				p.buf.WriteString(pl.Name)
 				p.buf.WriteRune('"')
 			}
-			if pl.FrameRate != 0.0 {
-				p.buf.WriteString(",FRAME-RATE=\"")
-				p.buf.WriteString(strconv.FormatFloat(pl.FrameRate, 'f', 3, 32))
-				p.buf.WriteRune('"')
-			}
 			if pl.VideoRange != "" {
-				p.buf.WriteString(",VIDEO-RANGE=\"")
+				p.buf.WriteString(",VIDEO-RANGE=")
```

Comment:
video-range and hdcp-level are enumerated strings, see example: https://developer.apple.com/documentation/http_live_streaming/hls_authoring_specification_for_apple_devices/hls_authoring_specification_for_apple_devices_appendices